src/lib/schematics/dashboard/index.spec.ts

```diff
@@ -47,4 +44,17 @@ describe('material-dashboard-schematic', () => {
       `import { MatGridListModule, MatCardModule, MatMenuModule, MatIconModule, MatButtonModule } from '@angular/material';`);
   });
 
+  it('should support passing the style extension option', () => {
+    const tree = runner.runSchematic('dashboard', {styleext: 'scss', ...baseOptions},
+      createTestApp());
```

Comment:
small nit: prefer line breaking at the higher syntactic level
```ts
    const tree = 
        runner.runSchematic('dashboard', {styleext: 'scss', ...baseOptions}, createTestApp());
```

Reply:
Makes sense - I've started the wrap after the function name. That's what we do within other Jasmine tests as well (e.g. `expectTo(`) and that's also what CLang proposed me to do.